rpc_jobs/rpc_openstack.yml

```diff
@@ -174,7 +174,6 @@
     branch: "master"
     jira_project_key: "RO"
     image:
-      - xenial_mnaio
```

Comment:
why was this removed?

Reply:
Strict artifact builds using MNAIO may or may not work depending on when upstream packages are released versus the last successful build because the MNAIO builds deploy the base OS using the latest upstream packages, not the artifact repo. As such, this test is a bit pointless until we get the base OS deployment working with the artifacts.

Reply:
ok